### PR TITLE
fast-track: Clarify the JavaSE support in Jakarta Rest Specification

### LINKED ISSUE #1127: Clarify that java Se support in Section 2.3
Currently Section 2.3 of the Jakarta RESTful Web Services Specification states the following:
`A compliant implementation MUST support both alternatives on Java SE.`

Via notes in the [rest-dev] mailing list I've confirmed that the consensus was that _"The Java SE Bootstrap API is mandatory for those compliant products that already provide a way to natively bootstrap on Java SE."_. [https://www.eclipse.org/lists/rest-dev/msg00008.html](https://www.eclipse.org/lists/rest-dev/msg00008.html)

Therefore I think that the statement above should be removed from section 2.3 and a new statement similar to the following be added to Section 2.3.1 Java SE:
```
There are two alternative ways of publishing on Java SE: Creating SE endpoints directly or using the SE bootstrap
API. Both are described in this section. Any compliant implementation that supports either of these alternatives on Java SE MUST support both.
```

### PR DESCRIPTION
**Pull request by jim-krueger**
Fixes #1127

### CODE REVIEW

**Review by jansupol — APPROVED** [review]
Sounds good

**Review by jamezp — DISMISSED** [review]

**Review by spericas — APPROVED** [review]

**Comment by mkarg** [comment]
#1127 is not a bug, so there is no reason to fast-track it.

File: jaxrs-spec/src/main/asciidoc/chapters/applications/_publication.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿////
 *******************************************************************
-* Copyright (c) 2019 Eclipse Foundation
+* Copyright (c) 2019, 2024 Eclipse Foundation
```
Comment (mkarg) [inline jaxrs-spec/src/main/asciidoc/chapters/applications/_publication.adoc:3]: Please remove this. It is not needed according to the EMO, and adding it adds unnecessary work in the future.

File: jaxrs-tck-docs/assertions/JAXRSSpecAssertions_3.1.0.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <META http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <!--
 
-    Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2021, 2024 Oracle and/or its affiliates. All rights reserved.
```
Comment (mkarg) [inline jaxrs-tck-docs/assertions/JAXRSSpecAssertions_3.1.0.html:6]: Same here.

**Review by mkarg — CHANGES_REQUESTED** [review]
Please do not add Copyright end dates if not already given.

File: jaxrs-spec/src/main/asciidoc/chapters/applications/_publication.adoc (hunk 1 above)
Comment (jim-krueger) [reply jaxrs-spec/src/main/asciidoc/chapters/applications/_publication.adoc:3]: Serves me right for assuming the copyright practices and restrictions would be universal.   I will certainly conform to the EMO (https://www.eclipse.org/projects/handbook/#ip-copyright-headers).  Thanks for pointing this out.

**Comment by jim-krueger** [comment]
@mkarg The changes you requested have been made,  please re-review.  Thanks

**Review by spericas — APPROVED** [review]

**Review by jansupol — APPROVED** [review]

**Comment by spericas** [comment]
@mkarg Can you please review it again?

**Review by spericas — APPROVED** [review]

File: jaxrs-spec/src/main/asciidoc/chapters/applications/_publication.adoc (hunk 1 above)
Comment (jansupol) [reply jaxrs-spec/src/main/asciidoc/chapters/applications/_publication.adoc]: From the same document:
> An Eclipse Foundation open source project is not a legal entity. Further, the Eclipse Foundation does not assume ownership of contributions (the Eclipse Foundation is only a copyright holder in those cases where an employee of the Eclipse Foundation has authored the contribution).

I understand it the Eclipse Foundation should not be the copyright holder, rather `Contributors to the Eclipse Foundation` should have been used.

**Review by jamezp — APPROVED** [review]

**Review by jansupol — APPROVED** [review]

**Review by mkarg — APPROVED** [review]

**Pull request merged**